README.md

```diff
@@ -1 +1,21 @@
 # OpenFisca new Web-API
+
+## Install
+
+```sh
+git clone git@github.com:openfisca/openfisca-web-api.git
```

Comment:
Always give `https` URLs. SSH-based ones are more prone to failing, as they depend on your user having a GitHub account.